packages/web3-eth-accounts/package.json

```diff
@@ -20,7 +20,7 @@
         "eth-lib": "0.2.8",
         "ethereumjs-util": "^7.0.10",
         "scrypt-js": "^3.0.1",
-        "uuid": "3.3.2",
+        "uuid": "8.3.2",
```

Comment:
That's a drastic change. Let's hope nothing breaks 🤞

Reply:
@nazarhussain Are there specific things you are concerned about that we could add test cases for? What do you think the biggest risk factors are?